CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - `tt install tarantool-dev`: ability to install tarantool from the local build directory.
 - `tt uninstall`: smart auto-completion. It shows installed versions of programs.
+- `tt pack --without-modules`: allows not to take external modules into the pack bundle.
```

Comment:
tt pack: added option `--without-modules` allowing not to take external modules into the pack bundle.